modules/references/references_test.go

```diff
@@ -43,10 +43,6 @@ func TestFindAllIssueReferences(t *testing.T) {
 				{29, "", "", "29", true, XRefActionCloses, &RefSpan{Start: 15, End: 18}, &RefSpan{Start: 7, End: 13}},
 			},
 		},
-		{
-			"#123 no, this is a title.",
```

Comment:
No it isn't a title - this was broken blackfriday behaviour and I don't think we should continue that.